docs: Focus indicator guidance (#1306)
cultureamp/kaizen-design-system: site/docs/guidelines/interaction-states.mdx

## To keep in mind

### Focus indicator

> - Any element a user can interact with must have a focus indicator when tabbing while using keyboard navigation. This helps users with screen readers and limited mobility to understand what they are interacting with and where they are on the page. 

Reviewer: This isn't really benefiting screen reader users (there's a good chance they won't be able to see a focus state!) but it helps with keyboard only users, voice control users, and other forms of assistive tech

```suggestion
- Any element a user can interact with must have a focus indicator when tabbing while using keyboard navigation. This helps users with assistive technology understand what they are interacting with and where they are on the page. 
```

- All input fields must include a focus indicator when active, whether using keyboard navigation or click interactions. This includes all types of [Form](/guidelines/forms) elements:
  - Text field
  - [Search field](/components/search-field)
  - [Checkbox](/components/checkbox-group) (on checkbox, exclude focus on label)
  - [Radio button](/components/radio-field) (on radio button, exclude focus on label)
  - [Select](/components/select)
  - [Date picker](/components/date-picker)

>   - [Dropdown](/components/date-picker)

Reviewer: ```suggestion
  - [Dropdown](/components/dropdown)
```

>   - [Likert Scal](/components/docs)

Reviewer: ```suggestion
  - [Likert Scale](/components/docs)
```

(Also... why is the URL for this called "docs" 😕)
